src/Sylius/Bundle/CoreBundle/spec/Doctrine/ORM/PaymentMethodRepositorySpec.php

```diff
@@ -1,57 +0,0 @@
-<?php
-
-namespace spec\Sylius\Bundle\CoreBundle\Doctrine\ORM;
-
-use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\ORM\EntityManager;
-use Doctrine\ORM\Mapping\ClassMetadata;
-use Doctrine\ORM\QueryBuilder;
-use PhpSpec\ObjectBehavior;
-use Prophecy\Argument;
-use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
-use Sylius\Component\Core\Model\ChannelInterface;
-use Sylius\Component\Payment\Model\PaymentMethodInterface;
-use Sylius\Component\Payment\Repository\PaymentMethodRepositoryInterface;
-
-class PaymentMethodRepositorySpec extends ObjectBehavior
```

Comment:
We decided to not spec repositories because of builder pattern used which is pain and in the end, it does not cover us that much from regressions etc. I'd prefer to cover these repositories with integration tests or just assume they work according to scenarios passing.